theme/material/vaadin-tabs-styles.html

```diff
@@ -76,8 +76,7 @@
       /* RTL specific styles */
 
       :host([dir="rtl"]) [part="forward-button"] {
-        left: auto;
-        right: 0;
+        left: 0;
```

Comment:
Shouldn't it be:
```suggestion
        right: 0;
```

Reply:
Hmmm, the `:host(:not([dir="rtl"])) [part="forward-button"]` is `right: 0;`, so this one should be `left: 0`

Reply:
Ah, okay. My bad.